docs/declaring-loaders.rst

```diff
@@ -5,22 +5,40 @@ Declaring Item Loaders
 
 Item Loaders are declared by using a class definition syntax. Here is an example::
 
-    from itemloaders import ItemLoader
-    from itemloaders.processors import TakeFirst, MapCompose, Join
+   from dataclasses import dataclass, field
 
-    class ProductLoader(ItemLoader):
+   from itemloaders import ItemLoader
+   from itemloaders.processors import TakeFirst
 
-        default_output_processor = TakeFirst()
 
-        name_in = MapCompose(str.title)
-        name_out = Join()
+   def price_out(values):
+      return float(values[0])
 
-        # using a built-in processor
-        price_in = MapCompose(str.strip)
 
-        # using a function
-        def price_out(self, values):
-            return float(values[0])
+   @dataclass(init=False)
+   class Product:
+      name: str
```

Comment:
3-space indent detected

Reply:
Not sure if you were talking about `'output_processor'`, but that's the wrong indentation I found, unless I'm missing something :thinking:

Reply:
The whole example seems to be using multiples of 3 for indentation.

Reply:
For some reason, my editor had the wrong config for this file :(
Also identified this problem in another and pushed the changes.
Nice catch!